core-application/build.gradle

```
@@ -16,6 +16,14 @@ dependencies {
     implementation 'org.springframework:spring-context'
     implementation 'com.fasterxml.jackson.core:jackson-annotations:2.15.2'
     implementation 'jakarta.validation:jakarta.validation-api'
+
+    testImplementation(testFixtures(project(":core-domain")))
+    testImplementation 'org.mockito:mockito-core:5.3.1'
+    testImplementation 'org.mockito:mockito-junit-jupiter:5.3.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.3'
+    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.9.3'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.3'
+    testImplementation "org.assertj:assertj-core:3.24.2"
```

```suggestion
    testImplementation(testFixtures(project(":core-domain")))
    
    testImplementation 'org.mockito:mockito-core:5.3.1'
    testImplementation 'org.mockito:mockito-junit-jupiter:5.3.1'
    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.3'
    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.9.3'
    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.3'
    testImplementation "org.assertj:assertj-core:3.24.2"
```

ㅎㅎ.. 위와 동일하게 가면 어떨까요..

---

공백라인 추가했습니다!!!